### PR TITLE
Develop react_person

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
[DEMO LINK](https://sergey-lipikhin.github.io/react_person/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```
Comment (valeriiayarzhemska) [inline src/App.jsx:3]: u can delete last 'Person'

**Review by valeriiayarzhemska — APPROVED** [review]
just one little comment:)

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:28]: if age = 0 I will see 0 :)

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,33 @@
...
+    ? 'wife'
+    : 'husband';
+
+  const marriageInfo = isMarried
+    ? `${partnerName} is my ${partnerStatus}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:22]: hard to read, why write everything in one line?

**Review by Andr1yK — CHANGES_REQUESTED** [review]

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,35 @@
...
+    ? `${partnerName} is my ${partnerStatus}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {!!age && (
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:26]: easier to read

```suggestion
      {Boolean(age) && (
```

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**